lib/standards/html-aria.js

```diff
@@ -0,0 +1,707 @@
+// Source: https://www.w3.org/TR/html-aria/#allowed-aria-roles-states-and-properties
+// Source: https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings
+const htmlAria = {
```

Comment:
I preferred this when it was "htmlElms", see: https://github.com/dequelabs/axe-core/pull/2337#pullrequestreview-440760860